t/commands/file/extract.t

```diff
@@ -0,0 +1,41 @@
+use strict;
+use warnings;
+
+use Cwd 'getcwd';
+my $cwd = getcwd;
+
+BEGIN {
+  use Test::More tests => 57;
+  use Data::Dumper;
+
+  use_ok 'Rex';
+  use_ok 'Rex::Commands::File';
+  use_ok 'Rex::Interface::Exec';
+  Rex::Commands::File->import;
+  Rex::Interface::Exec;->import;
+}
+
+my $tmp_dir = "/tmp";
+if ( $^O =~ m/^MSWin/ ) {
+  $tmp_dir = $ENV{TMP};
+}
+
+my $filename = "${tmp_dir}/test-$$.tgz";
+my $to = "myfile_extracted/";
+
+my $exec = Rex::Interface::Exec->create;
+my $cmd = "tar cfvz ${filename} /usr/local/bin/";
+$exec->exec($cmd);
```

Comment:
Can it be that a simple `run` command would be adequate here instead of the dance with `Rex::Interface::Exec`?